apis/v1alpha1/observabilitypolicy_types.go

```diff
@@ -7,11 +7,12 @@ import (
 
 // +genclient
 // +kubebuilder:object:root=true
-// +kubebuilder:storageversion
+// +kubebuilder:deprecatedversion:warning="The 'v1alpha1' version of ObservabilityPolicy API is deprecated, please migrate to 'v1alpha2'."
```

Comment:
Talked offline with @sjberman and we came to the conclusion that this change does not require a version bump. With our current validation, if targetRefs is omitted or empty, the policy will be rejected. Adding a min length of 1 to the list of targetRefs will only change the error message, not the behavior.